cmd/registry/cmd/apply/testdata/manifest.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: apigeeregistry/v1
```

Comment:
Is the `id` field in manifest replaced by `name`? `id` was used before to name the artifact stored in the registry.
Will this configuration create an artifact named `projects/demo/location/global/artifacts/manifest`? 

We will need some code change to take `name` instead of `id` while creating the manifest.

I saw somewhere down in the code for `upload artifact` that the artifactName is set using `header.id` field. Should we make `id` a required field in these yaml definitions?

Reply:
This is following the [Kubernetes object format](https://kubernetes.io/docs/concepts/overview/working-with-objects/kubernetes-objects/), which uses `name` in the way that we use `id`, and I think it's better to be consistent with Kubernetes than to make the input format match all of the definitions associated with the AIPs.

I think these functions might subsume `upload artifact`, and if so, we might want to deprecate the `id` and `kind` fields that were added to support `upload artifact`.